### PR TITLE
First PR

### PR DESCRIPTION
**Pull request by dormullor**

### CODE REVIEW

**Comment by dormullor** [comment]
@jbw976

File: OWNERS.md
Comment (jbw976) [inline OWNERS.md]: Do you want to update this list with the folks that you intend to be maintaining this repo going forward? we can then get the permissions set up to give them maintainer perms :)

File: README copy.md
Comment (jbw976) [inline README copy.md]: do you want to add some useful content, e.g. a quick start that tells users how to use this repo/provider into a `README.md`? That could give a lot of useful context to help people be successful on their first try with this project.

File: examples/provider/config.yaml
Comment (jbw976) [inline examples/provider/config.yaml:5]: are there further details that could be added into this ProviderConfig example that would help people know how to use it?

**Review by jbw976 — COMMENTED** [review]
Thank you very much for making this contribution @dormullor! If it's possible, having the readme/examples fleshed out just a bit more would be useful so the rest of the community can understand and start to either use or contribute to this provider too.  What do you think about doing that in this first PR, or would you like to defer to a 2nd PR after this repo is bootstrapped?

File: examples/provider/config.yaml
Comment (dormullor) [reply examples/provider/config.yaml:5]: Currently, the providerConfig should remain empty. i wanted to keep it for adding capabilities in the future

File: OWNERS.md
Comment (jbw976) [inline OWNERS.md:13]: is this the right github account you want to use as a maintainer? it's different than the account that this PR was authored with, so I wanted to make sure we're not giving access to the wrong folks :)

**Review by jbw976 — APPROVED** [review]
LGTM @dormullor - we can go ahead and merge this so the repo has this content, but please do let me know about the maintainer list (is @dorroddorrod the right account for maintainer?) so we can get permissions set up going forward on this repo :)

**Pull request merged**
